common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -198,6 +189,15 @@ def get_transcripts_for_student(self, transcripts):
         return track_url, transcript_language, sorted_languages
 
     def get_html(self):
+        try:
+            import edxval.api as edxval_api
+        except ImportError:
+            edxval_api = None
+        try:
+            from branding.models import BrandingInfoConfig
+        except ImportError:
+            BrandingInfoConfig = None
```

Comment:
I'm not proud of sprinkling these try/except import blocks in multiple places. But the comment at the top at least tells why. Oh - haven't added comments about these import moves yet. I'll do so.

Reply:
Would it be worth creating helper functions for this, like `edxval_api = import_edxval_api()`?  Could even make it just `import_edxval_api()` if the function directly adds to `globals()` the first time and just looks it up thereafter, although pylint and/or PyCharm might be less happy about that.

Reply:
I've pushed up a version of this suggestion for your review.